.github/workflows/cleanup_base.yaml

```
@@ -18,7 +18,7 @@ jobs:
 
     steps:
       - name: Cleanup old images
-        uses: SmartsquareGmbH/delete-old-packages@v0.7.0
+        uses: SmartsquareGmbH/delete-old-packages@v0.8
```

```suggestion
        uses: SmartsquareGmbH/delete-old-packages@v0.8.1
```

It seems that the current policy is to fix also the minor version